loader.js

```diff
@@ -62,11 +60,10 @@ module.exports = function() {
   })
     .then(() => {
       const pkg = require(join(pkgDir, "./package.json"));
-
+      const exportPath = join(pkgDir, pkg.main).replace(/\\/g, '/');
```

Comment:
Just to make sure I understand, the replace fixes the path in pkg.main?

Reply:
the `replace` fixes the path in the result of joining `pkgDir` and `pkg.main`. In Windows, it will be joined using `\\` so this will error when trying to access the file later on.

Reply:
The join is the expected behavior, I'm just wondering why do you need to turn it back to non-windows patch again? Does Webpack fail?

Reply:
yes, because you're using the generated string to create the `export` which does not support Windows path style.